config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_backend_trace_attributes.conf

```diff
@@ -171,6 +171,15 @@ commands = [
               scope: BACKEND_TRACE,
               sources: [QS],
               type: ATTRIBUTE
+            },
+            {
+              fqn: Backend.Trace.backend_operation,
+              key: backendOperation,
```

Comment:
can we shorten this key to just `operation` ? the scope already indicates the "backend" part of it.

Reply:
Done